### DISCUSSION TITLE
tutorials/growing-graphql-api-dgs-lab/03-artist-datasource

### DISCUSSION THREAD

**Discussion by giscus** [apollographql-education/content-feedback#113] [course feedback]
# tutorials/growing-graphql-api-dgs-lab/03-artist-datasource

https://www.apollographql.com/tutorials/growing-graphql-api-dgs-lab/03-artist-datasource

**Comment by mkatsman** [comment]
How do I ty it out if I do not have an artist ID?  both artist apis require artist Id.

**Reply by lizhennessy** [reply]
Ah great catch! Ill be sure to provide one!

**Comment by mkatsman** [comment]
What is the sandbox input value?

**Reply by lizhennessy** [reply]
Hi there @mkatsman ! do you mean what do we use to connect to Sandbox (i.e. what address our server's running on)? Or are you curious about the kind of query that we'd send through Sandbox to test out our artist method implementation?

**Reply by mkatsman** [reply]
I do connect to the sandbox, no problem there. To execute the artist end point as it says in the assignment,  I need an artist Id input value. I now see, that it is there. I did not see it before. Thank you.

**Reply by mkatsman** [reply]
different unexpected problem. I am unable to copy the artistId from that check box line. It just does not allow me to do that.

**Comment by mkatsman** [comment]
It would probably be good to have a sandbox request here. You try this, and it works.

**Comment by mkatsman** [comment]
the response for followers from spotify, looks like this:    "followers": {
    "href": null,
    "total": 3697686
  }, that does not match our definition of Int.

type Artist {
  "The ID for the artist."
  id: ID!
  "The name of the artist"
  name: String
  "The number of followers the artist has"
  followers: Int
  "A collection of genres that can be ascribed to this artist"
  genres: [String]
  "The URI for the artist, usually a Spotify link"
  uri: String
}
